Doc/library/enum.rst

```diff
@@ -281,7 +281,7 @@ Iterating over the members of an enum does not provide the aliases::
     >>> list(Shape)
     [<Shape.SQUARE: 2>, <Shape.DIAMOND: 1>, <Shape.CIRCLE: 3>]
 
-The special attribute ``__members__`` is an ordered dictionary mapping names
+The special attribute ``__members__`` is a dictionary mapping names
```

Comment:
I'm not comfortable with removing the word "ordered".  I wonder if we should add "read-only"?

`is a read-only ordered dictionary ...`